src/ResourceManager/Commerce/Commands.UsageAggregates/GetUsageAggregatesCommand.cs

```diff
@@ -0,0 +1,70 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+
+using Microsoft.Azure.Commerce.UsageAggregates;
+using Microsoft.Azure.Common.Authentication;
+using Microsoft.Azure.Common.Authentication.Models;
+
+namespace Microsoft.Azure.Commands.UsageAggregates
+{
+    using Commerce.UsageAggregates.Models;
+    using WindowsAzure.Commands.Utilities.Common;
+    using System;
+    using System.Management.Automation;
+
+    [Cmdlet(VerbsCommon.Get, "UsageAggregates"), OutputType(typeof(UsageAggregationGetResponse))]
+    public class GetUsageAggregatesCommand : AzurePSCmdlet
+    {
+        [Parameter(Mandatory = true, HelpMessage = "The start of the time range to retrieve data for.")]
+        public DateTime ReportedStartTime { get; set; }
+
+        [Parameter(Mandatory = true, HelpMessage = "The end of the time range to retrieve data for.")]
+        public DateTime ReportedEndTime { get; set; }
+
+        private AggregationGranularity aggregationGranularity = AggregationGranularity.Daily;
+        [Parameter(Mandatory = false, HelpMessage = "Value is either daily (default) or hourly to tell the API how to return the results grouped by day or hour.")]
+        public AggregationGranularity AggregationGranularity { 
+            get { return aggregationGranularity; } 
+            set { aggregationGranularity = value;}
+        }
+
+        private bool showDetails = true;
```

Comment:
Please use _ for private fields, and include them in the section above the public properties